### PR TITLE
Linux disk collector

### PR DESCRIPTION
**Pull request by Kicer86**

### CODE REVIEW

File: src/agent/linux/LsblkOutputParser.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class LsblkOutputParser
+{
+public:
+    struct LsblkEntry
+    {
+        QString name;
+        int major;
+        int minor;
+        int rm;
```
Comment (wkucjas) [inline src/agent/linux/LsblkOutputParser.h]: Not descriptive name

Comment (wkucjas) [inline src/agent/linux/LsblkOutputParser.h]: Not descriptive name

File: src/agent/linux/LsblkOutputParser.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        QString size;
+        bool ro;
```
Comment (wkucjas) [inline src/agent/linux/LsblkOutputParser.h]: Not descriptive name

File: src/common/OutputParsersUtils.cpp [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+#include "OutputParsersUtils.h"
+
+namespace ParsersUtils
+{
+
+    namespace
+    {
+        QStringList cleanup(const QStringList& list)
```
Comment (wkucjas) [inline src/common/OutputParsersUtils.cpp:9]: Is it necessary using QT types, not STD ones?

File: src/agent/linux/LsblkOutputParser.cpp [hunk 4]
```diff
@@ -1,9 +1,89 @@
...
+        QString type;
+        std::uint64_t size;
+        int major;
+        int minor;
+    };
+
+    RawLsblkEntry parseLine(const QString& line)
+    {
+        const QStringList cols = line.split(' ');
+        assert(cols.size() >= 6);
```
Comment (wkucjas) [inline src/agent/linux/LsblkOutputParser.cpp]: What does magic number 6 mean?

File: src/agent/linux/LsblkOutputParser.cpp [hunk 5]
```diff
@@ -1,9 +1,89 @@
...
+
+        const QStringList major_minor = cols[1].split(':');
+        assert(major_minor.size() == 2);
```
Comment (wkucjas) [inline src/agent/linux/LsblkOutputParser.cpp]: Magic number 2?

File: src/agent/linux/LinuxDiskCollector.cpp [hunk 6]
```diff
@@ -16,7 +19,47 @@ std::vector<Disk> LinuxDiskCollector::GetDisksList()
...
+    return disks;
+}
+
+
+QString LinuxDiskCollector::modelFor(const QString& device) const
+{
+    const QString modelNumber = "Model Number:";
+
+    QProcess hdparm;
+    hdparm.start("hdparm", { "-I", QString("/dev/%1").arg(device) }, QProcess::ReadOnly);
```
Comment (wkucjas) [inline src/agent/linux/LinuxDiskCollector.cpp]: What -I means? Maybe comment will be good for explanation.

File: src/agent/linux/LsblkOutputParser.h (hunk 1 above)
Comment (Kicer86) [reply src/agent/linux/LsblkOutputParser.h]: it's outdated

File: src/agent/linux/LsblkOutputParser.h (hunk 2 above)
Comment (Kicer86) [reply src/agent/linux/LsblkOutputParser.h]: it's outdated

File: src/common/OutputParsersUtils.cpp (hunk 3 above)
Comment (Kicer86) [reply src/common/OutputParsersUtils.cpp:9]: Qt's containers have more useful stuff + this function will cooperate with qt code

**Review by wkucjas — APPROVED** [review]

**Pull request merged**